### PR TITLE
Symbol or Abbreviation instead of initial

### PR DESCRIPTION
**Pull request by aminya**
This PR replaces initial with Symbol (icon) or Abbreviation

### CODE REVIEW

**Comment by aminya** [comment]
@lierdakil I tried to use your fonts like your SematicsView, but I cannot make it work. Do you have an idea why?

File: src/outlineView.ts [hunk 1]
```diff
@@ -135,6 +135,19 @@ const supportedTypes = [
   "variable",
 ];
 
+const symbolMap = {
+  class: '\\e600',
```
Comment (aminya) [inline src/outlineView.ts:139]: I tried without escaping too
```suggestion
  class: '\e600',
```

**Comment by lierdakil** [comment]
Probably because you're not in CSS, so CSS Unicode escapes don't work? Use JS Unicode escapes, e.g. `\ue600` (notice the `u`).

**Comment by aminya** [comment]
> Probably because you're not in CSS, so CSS Unicode escapes don't work? Use JS Unicode escapes, e.g. `\ue600` (notice the `u`).

Thanks! Now it works for most of the symbols. Do you have the source for these fonts? I want to add symbols for all of the supported kinds.

**Comment by lierdakil** [comment]
I don't. That font was created by @basarat I think, way back when he was spearheading TypeScript support in Atom.

That said, it's just a regular vector font. Fontforge is fully capable of working with it.

**Pull request merged**